python/sglang/srt/layers/sampler.py

```diff
@@ -50,6 +50,20 @@ def forward(
         else:
             # Post process logits
             logits.div_(sampling_info.temperatures)
+
+            if any(sampling_info.top_n_sigmas > 0):
```

Comment:
move this check to `sampling_batch_info.py` and name it `need_top_n_sigmas_sampling`, similar to how we do `need_min_p_sampling`.